third_party/purge/helpers/varnish_helper.php

```diff
@@ -18,11 +18,13 @@ function send_purge_request($site_url = NULL, $site_port = NULL)
 		}
 		else
 		{
-			$parsed_url = parse_url($site_url);
+			$purge_url = $site_url;
+			$port = $site_port;
+			/*$parsed_url = parse_url($site_url);
 			$url_path = isset($parsed_url['path']) ? $parsed_url['path'] : '/';
 			$url_scheme = isset($parsed_url['scheme']) ? $parsed_url['scheme'] . '://' : $protocol;
 			$purge_url = $url_scheme . $parsed_url['host'] . $url_path;
-			$port = ( ! isset($parsed_url['port']) || empty($parsed_url['port'])) ? 80 : $parsed_url['port'];
+			$port = ( ! isset($parsed_url['port']) || empty($parsed_url['port'])) ? 80 : $parsed_url['port'];*/
```

Comment:
You can delete code instead of commenting it out, that's what source control is for.